dspace-spring-rest/src/main/java/org/dspace/app/rest/configuration/RESTSpringLoader.java

```diff
@@ -15,6 +15,8 @@
 
 /**
  * Utility class that will load the Spring XML configuration files related to the REST webapp
+ *
+ * @author Atmire NV (info at atmire dot com)
```

Comment:
While we haven't really "policed" `@author` tags closely in recent years, I feel we should likely keep them pointing at individuals (who want credit for creating/refactoring specific classes) instead of organizations. I'm a bit worried that labeling classes with organizational contact info is a bit misleading, as it no longer refers to the specific creator of the class, and also could become an advertising gimmick/battle within our codebase (over which company gets more credit). The only company that "owns" the code is DuraSpace, and we just own the copyright in order to ensure the code remains open source, etc.

So, personally, I feel we should avoid this sort of change in our codebase. Either leave off the `@author` altogether, or reference the individual(s) who created / refactored the class.

However, if others disagree strongly here, I'll gladly bring this to discussion within the Committers group and/or Steering Group, so that we can formalize a policy on `@author` tags